package.json

```diff
@@ -13,24 +13,29 @@
   },
   "author": "winkerVSbecks",
   "license": "MIT",
-  "main": "dist/preset",
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
+  "types": "dist/ts/index.d.ts",
   "files": [
     "dist/**/*",
     "README.md",
     "*.js"
   ],
   "scripts": {
     "clean": "rimraf ./dist",
-    "buildBabel": "babel ./src --out-dir ./dist --extensions \".js,.jsx,.ts,.tsx\"",
-    "buildTsc": "tsc --declaration --emitDeclarationOnly --outDir ./dist",
-    "build": "concurrently \"yarn run buildBabel\" \"yarn run buildTsc\"",
-    "build:watch": "concurrently \"yarn run buildBabel -- --watch\" \"yarn run buildTsc -- --watch\"",
+    "buildBabel": "concurrently \"yarn buildBabel:cjs\" \"yarn buildBabel:esm\" \"yarn buildTsc\"",
```

Comment:
It doesn't seem like `buildTsc` belongs in `buildBabel`, does it?  I think that's handled by `build` below.

Reply:
Good catch!